### ISSUE TITLE
KOTS Admin Console should support running under a subpath

### ISSUE THREAD

**Issue by nathan-mellis** [replicatedhq/kots#646]
I would like to be able to expose the KOTS admin console under a subpath of the main domain that hosts my application. Something like https://path.to.product/admin-console, but the front-end is currently coded so that all links to CSS, JavaScript, etc. are prefixed with a slash (/) so that they always request from the root of the domain.

Supporting this also helps solve the issue I filed in ticket #645. It would provide a way to expose the admin console using the existing application's DNS and TLS.

**Comment by dexhorthy** [comment]
cc @marccampbell

**Comment by dexhorthy** [comment]
Questions so far for kots team: Does this get configured by a kots CLI flag? Or pulled from a configmap? Or set in kind: Application YAML? or kURL spec?

**Comment by divolgin** [comment]
@nathan-mellis can you provide some details about how traffic is routed to your application?  we are trying to figure out what options are available to us for routing requests.

**Comment by nathan-mellis** [comment]
@divolgin we are (or will be) using standard Kubernetes Ingress to route a HTTP subpath to a Kubernetes service.

**Comment by divolgin** [comment]
@nathan-mellis is this the proxy setup by kURL or something you will be configuring?  Also @dexhorthy , if you have any design ideas, feel free to share.

**Comment by nathan-mellis** [comment]
@divolgin No. We're not using kURL yet. This is the completely standard NGINX Ingress Controller: https://kubernetes.github.io/ingress-nginx/

**Comment by chitender** [comment]
just checkin in, is there any update on this?

**Comment by ddvdozuki** [comment]
Is there any update to this? I have a need to route a TLS terminated nginx ingress controller to the admin dashboard but for the life of me i cannot figure out how to tell kotsadm to append `/dashboard` to the urls it generates.
